tests/console/libmicrohttpd.pm

```diff
@@ -20,7 +20,7 @@ sub run {
     select_console 'root-console';
 
     # Install greenbone-security-assistant
-    zypper_call("in greenbone-security-assistant");
+    zypper_call("--no-gpg-checks in greenbone-security-assistant");
```

Comment:
Same as above, it is not new test case relevant. Please create separate commit for that.

Reply:
The same answer as top :)